plugins/GrapesJsBuilderBundle/package.json

```diff
@@ -26,20 +40,34 @@
   },
   "homepage": "https://github.com/mautic/plugin-grapesjs-builder#readme",
   "dependencies": {
-    "grapesjs": "^0.17.29",
-    "grapesjs-mjml": "^0.5.8",
+    "grapesjs": "^0.21.8",
+    "grapesjs-blocks-basic": "^1.0.1",
+    "grapesjs-component-countdown": "^1.0.1",
+    "grapesjs-custom-code": "^1.0.1",
+    "grapesjs-mjml": "^1.0.5",
+    "grapesjs-navbar": "^1.0.2",
     "grapesjs-parser-postcss": "^0.1.1",
+    "grapesjs-plugin-ckeditor": "^0.0.10",
     "grapesjs-plugin-ckeditor5": "^0.0.33",
-    "grapesjs-preset-mautic": "github:mautic/grapesjs-preset-mautic#ckeditor5",
-    "grapesjs-preset-newsletter": "^0.2.21",
-    "grapesjs-preset-webpage": "^0.1.11"
+    "grapesjs-plugin-export": "^1.0.11",
+    "grapesjs-plugin-forms": "^2.0.5",
+    "grapesjs-preset-mautic": "github:LordRembo/grapesjs-preset-mautic#update-grapesjs-compatibility-m5",
```

Comment:
Is this intentional? Looks like you link it to your fork.

Reply:
@adiux Yes, as I explain in the intro of this ticket, there is a PR for this dependency that needs to be merged so I can remove my fork. That's why this PR is kept as Draft.